corehq/pillows/domain.py

```diff
@@ -61,3 +61,20 @@ def change_transform(self, doc_dict):
         for country in countries:
             doc_ret['deployment']['countries'].append(COUNTRIES[country].upper())
         return doc_ret
+
+
+def get_domain_reindexer():
+    pillow = DomainPillow(online=False)
+    return ElasticPillowReindexer(
+        pillow=pillow,
+        change_provider=CouchViewChangeProvider(
+            document_class=Domain,
+            view_name='all_docs/by_doc_type',
+            view_kwargs={
+                'startkey': ['Domain'],
+                'endkey': ['Domain', {}],
+            }
+        ),
+        elasticsearch=pillow.get_es_new(),
```

Comment:
Is it necessary to use the pillow to get a handle on the ES client? They all use the same client so seems like this could just be `get_es_new()` or even pushed down into the `ElasticPillowReindexer`